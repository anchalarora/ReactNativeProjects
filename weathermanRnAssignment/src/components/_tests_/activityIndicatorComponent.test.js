import React from 'react'
import renderer from 'react-test-renderer'

import ActivityIndicatorComponent from '../activityIndicatorComponent'


it('should load Activity Indicator without crashing', () => {
    const rendered = renderer.create(<ActivityIndicatorComponent />).toJSON()
    expect(rendered).toBeTruthy()
})


it('should load Activity Indicator without crashing', () => {
    const rendered = renderer.create(<ActivityIndicatorComponent />).toJSON()
    expect(rendered).toMatchSnapshot()
})