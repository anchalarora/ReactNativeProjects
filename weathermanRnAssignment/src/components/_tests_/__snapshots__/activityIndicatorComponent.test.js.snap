// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should load Activity Indicator without crashing 1`] = `
<ActivityIndicator
  animating={true}
  color="#FFFFFF"
  hidesWhenStopped={true}
  size="large"
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "space-around",
      "marginTop": 50,
      "padding": 10,
    }
  }
/>
`;
